torch/jit/_isinstance.py

```diff
@@ -0,0 +1,107 @@
+from typing import List, Dict, Tuple, Union, Optional
+
+
+def get_origin(target_type):
+    return getattr(target_type, "__origin__", None)
+
+
+def get_args(target_type):
+    return getattr(target_type, "__args__", None)
+
+
+def check_args_exist(target_type):
+    if target_type is List or target_type is list:
+        raise RuntimeError(
+            "Attempted to use List without a "
+            "contained type. Please add a contained type, e.g. "
+            "List[int]"
```

Comment:
Consider creating a template string for this and reusing it. All of these error messages look very similar except for type names.

Reply:
Is there a strong reason to create a template string? I don't think this version of the explicit string is bad